packages/v3-sdk/src/functions/getTokensPrice.ts

```diff
@@ -0,0 +1,21 @@
+import { BigintIsh, Price, Token } from '@pancakeswap/sdk'
+import { Q192 } from '../internalConstants'
+
+/**
+ * Returns the current price of the given tokens
+ *
+ * @param tokenA
+ * @param tokenB
+ * @param currentSqrtRatioX96
+ * @returns [token0/token1, token1/token0]
+ */
+export const getTokensPrice = (
```

Comment:
suggest we split into two separate functions, since normally we only need either of them instead of both

Reply:
like `getToken0Price` and `getToken1Price`

Reply:
also let's use sorted token as input for better performance. Since we use this function under the context of v3 pool, there's no need to sort token for each call